project/Project.scala

```diff
@@ -76,11 +76,7 @@ object Zipkin extends Build {
     crossPaths := false,            /* Removes Scala version from artifact name */
     fork := true, // forking prevents runaway thread pollution of sbt
     baseDirectory in run := file(cwd), // necessary for forking
-    resolvers := Seq(
-      Resolver.jcenterRepo,
-      Resolver.typesafeRepo("releases"),
-      "Twitter Maven Repository" at "https://maven.twttr.com/" // for thrift 0.5
-    ),
+    resolvers += "twitter-repo" at "http://maven.twttr.com", // for thrift 0.5
```

Comment:
Typesafe isn't needed. Jcenter isn't either except for "performance", but I
think it would come last so wouldn't help there anyway. Resolver order is
hard to control unless you set a JVM arg to override. I recommend leaving
this as is until gradle.. SG?